msom/src/main/java/pl/edu/uj/fais/wpz/msom/entities/Task.java

```diff
@@ -21,7 +22,7 @@
 
     private String name;
 
-    @ManyToOne
+    @ManyToOne(cascade = {CascadeType.ALL})
```

Comment:
Możesz u siebie lokalnie to usunąć i zobaczyc, czy działa? Bo głupio mi w pracy robić prywatne projekty xD